trunner/harness/unity.py

```diff
@@ -15,9 +15,12 @@ def unity_harness(dut: Dut, ctx: TestContext, result: TestResult) -> Optional[Te
     last_assertion = {}
     stats = {"FAIL": 0, "IGNORE": 0, "PASS": 0}
     results = []
+    timeout_val = 30
+    if ctx.nightly:
+        timeout_val = 100
```

Comment:
I think 100 is too much for one test case even for nightly tests - still they should be rather short. I think setting it to 60 would be absolutely fine.